src/components/faceVerification/screens/IntroScreen.jsx

```diff
@@ -176,21 +228,23 @@ const IntroScreen = ({ styles, screenProps, navigation }) => {
 
   useFVLoginInfoCheck(navigation)
 
-  useEffect(() => {
-    if (isFVFlow && isFVFlowReady && !disposing && enrollmentIdentifier) {
-      handleVerifyClick()
-    }
-  }, [isFVFlow, isFVFlowReady, disposing, enrollmentIdentifier])
+  if (state === 'pending') {
+    return (
+      <View display="flex" justifyContent="center">
+        <ActivityIndicator size="large" />
+      </View>
+    )
+  }
 
-  if (isFVFlow) {
+  if (expiryDate?.formattedExpiryTimestamp) {
```

Comment:
formatted is for display it shouldnt be used as an indicator.
not sure why the hook needs to return anything else but a timestamp
use the actual timestamp which is 0 or >0, make sure it is a number

Reply:
I just utilized an existing hook which is used in the goodid flow, because I did not see a point of having a separate hook for just the last-authenticated.
The hook is part of the goodid PR on the mono repo: https://github.com/GoodDollar/GoodWeb3-Mono/pull/128/commits/1985097e96fe0d42788c2eaf81d3a2925c586650

for clarity it now also exports lastAuthenticated to be used.

Why I did not do this before is because the .isZero() is already done there.

Reply:
applied fix here: https://github.com/GoodDollar/GoodDAPP/pull/4303/commits/90a8e5b39654c75a404d3bf885b7082d2165e811

Reply:
it should be expiryDate.lastAuthenticated, the first result in the array is an object

Reply:
ye mistake, fixed